docs/source/references/configurations.rst

```diff
@@ -11,23 +11,29 @@ These items are determined by the following order:
++=====================================+=======================================+=========+============================================+=================================================+
+| home_dir                            | LineaPy base folder                   | Path    | `$HOME/.lineapy`                           | `LINEAPY_HOME_DIR`                              |
++-------------------------------------+---------------------------------------+---------+--------------------------------------------+-------------------------------------------------+
+| artifact_storage_dir                | artifact saving folder                | Path    | `$LINEAPY_HOME_DIR/linea_pickles`          | `LINEAPY_ARTIFACT_STORAGE_DIR`                  |
++-------------------------------------+---------------------------------------+---------+--------------------------------------------+-------------------------------------------------+
+| database_url                        | LineaPy db connection string          | string  | `sqlite:///$LINEAPY_HOME_DIR/db.sqlite`    | `LINEAPY_DATABASE_URL`                          |
++-------------------------------------+---------------------------------------+---------+--------------------------------------------+-------------------------------------------------+
+| customized_annotation_folder        | user annotations folder               | Path    | `$LINEAPY_HOME_DIR/customized_annotations` | `LINEAPY_CUSTOMIZED_ANNOTATION_FOLDER`          |
++-------------------------------------+---------------------------------------+---------+--------------------------------------------+-------------------------------------------------+
+| do_not_track                        | disable user analytics                | boolean | false                                      | `LINEAPY_DO_NOT_TRACK`                          |
++-------------------------------------+---------------------------------------+---------+--------------------------------------------+-------------------------------------------------+
+| logging_level                       | logging level                         | string  | INFO                                       | `LINEAPY_LOGGING_LEVEL`                         |
++-------------------------------------+---------------------------------------+---------+--------------------------------------------+-------------------------------------------------+
+| logging_file                        | logging file path                     | Path    | `$LINEAPY_HOME_DIR/lineapy.log`            | `LINEAPY_LOGGING_FILE`                          |
++-------------------------------------+---------------------------------------+---------+--------------------------------------------+-------------------------------------------------+
+| mlflow_tracking_uri                 | mlflow tracking                       | string  | None                                       | `LINEAPY_MLFLOW_TRACKING_URI`                   |
```

Comment:
Maybe for documentation we should break down "integration specific" configurations into a separate section to prevent this table from becoming massive.